k8s.tf

```diff
@@ -49,6 +49,9 @@ resource "kubernetes_deployment_v1" "waggle_dance" {
           name = local.instance_alias
         }
         annotations = {
+          "ad.datadoghq.com/waggledance.check_names": "[\"openmetrics\"]"
+          "ad.datadoghq.com/waggledance.init_configs": "[{}]"
+          "ad.datadoghq.com/waggledance.instances": "[{ \"prometheus_url\": \"http://%%host%%:18000/actuator/prometheus\", \"namespace\": \"waggledance\", \"metrics\": [ \"metastore_status_total*\", \"counter_com_hotels_bdp_waggledance_server*\" ], \"send_histograms_buckets\": true,\"send_monotonic_counter\": true,\"send_distribution_buckets\": true,\"send_distribution_counts_as_monotonic\": true }]"
```

Comment:
I would prefer to have the metrics in a variable because you can have a metric per line. E.g.:
```
metrics = [
  metric1,
  metric2,
  ...,
  metricN
]
```